src/aind_data_upload_utils/check_metadata_job.py

```diff
@@ -0,0 +1,155 @@
+"""
+Module to check that certain metadata files exist and are in valid format.
+"""
+
+import argparse
+import json
+import logging
+import os
+import sys
+from pathlib import Path
+from time import time
+
+from pydantic import Field
+from pydantic_settings import BaseSettings
+
+# Set log level from env var
+LOG_LEVEL = os.getenv("LOG_LEVEL", "WARNING")
+logging.basicConfig(level=LOG_LEVEL)
+
+
+class JobSettings(BaseSettings):
+    """Job settings for CheckMetadataJob"""
+
+    metadata_dir: Path = Field(
```

Comment:
To avoid issues with how paths are represented in different operating systems, we can leave this as a string or a Union[Path, str]